aten/src/ATen/native/cuda/ForeachUnaryOp.cu

```diff
@@ -349,4 +352,58 @@ void foreach_tensor_round_cuda_(TensorList tensors) {
     foreach_unary_op_<Round>(tensors);
 }
 
+template<typename T>
+struct Trunc {
+    __device__ T operator()(T t) const { return std::trunc(t); }
+};
+
+std::vector<Tensor> foreach_tensor_frac_cuda(TensorList tensors) {
+    check_foreach_api_restrictions(tensors);
```

Comment:
The c++ style guide that most of the codebase follows uses 2-space indents. But as long as everything's consistent 4-space indents seem fine

Reply:
Yeah, everything is consistent